src/deluge/modulation/params/param_manager.cpp

```diff
@@ -30,12 +30,12 @@
 #include <new>
 
 ParamManager::ParamManager() {
-	summaries[0] = {0};
+	summaries[0] = {nullptr};
```

Comment:
This needs to be zero - it's using the gcc extension that treats {0} as initializing all members to zero (including two uint32s)

Reply:
Or {nullptr, 0, 0}

Reply:
fixed